routes/project/location.rb

```diff
@@ -3,7 +3,7 @@
 class Clover
   hash_branch(:project_prefix, "location") do |r|
     r.on String do |location_display_name|
-      @location = LocationNameConverter.to_internal_name(location_display_name)
+      @location = Location[display_name: location_display_name]
```

Comment:
```suggestion
      next unless (@location = Location[display_name: location_display_name])
```

Looks like this was an issue previously, but I assume we don't want to continue processing the request if an invalid location is used.

Note that once you support project-specific locations, you will need to make sure to only allow a location to be selected if it is for all projects or for this specific project.

Other routes should be checked to see we should also block invalid locations of those.